exercises/spiral-matrix/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+pub fn get_spiral(size: usize) -> Vec<Vec<usize>> {
```

Comment:
I'm not entirely certain that `get_spiral` is the best name. People will need to either use `spiral_matrix::get_spiral`, which is repetitive, or `get_spiral` alone, which is vague, and also falsely implies that the spiral is being fetched from somewhere instead of being generated.

Possibly just naming the function `spiral_matrix` would be best.